lib/Yield/PTML.cpp

```diff
@@ -431,7 +522,16 @@ std::string yield::ptml::controlFlowNode(const ::ptml::MarkupBuilder &B,
   auto Iterator = Function.Blocks().find(BasicBlock);
   revng_assert(Iterator != Function.Blocks().end());
 
-  auto Result = labeledBlock<false>(B, *Iterator, Function, Binary);
+  const model::Function &MFunction = Binary.Functions().at(Function.Entry());
+
+  // TODO: enable comments in the CFG too after the size computation rework.
```

Comment:
What do you mean here?

Did you consider pushing the injection of comments at `process-assembly` level?

Reply:
I don't remember, can you provide some context with specific examples?